tools/update/main.ts

```diff
@@ -331,7 +336,7 @@ class Updater {
         const dir = await mkdtemp(path.join(os.tmpdir(), `${owner}-${repoName}`))
 
         console.log(`Cloning ${fullRepoName} ...`)
-        await exec(`git clone git@github.com:${fullRepoName} ${dir}`)
```

Comment:
This assumes GitHub ssh is pre-configured. Switched to the HTTPS API for simplicity.

Reply:
I see the problem, though hard-coding HTTPS assumes that that is configured, but users usually only have one or the other configured. Maybe make this a CLI option? Happy that it defaults to HTTPS, I guess that is more common?

Reply:
hmm yeah, making it configurable sounds good!

Reply:
made it configurable: 6faf6765a0c6a818be02729cc7108f9aef637138